core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/db/dao/FileStatusDao.scala

```diff
@@ -9,7 +9,7 @@ import uk.gov.nationalarchives.tdr.api.core.db.dao.ConsignmentDao.consignments
 import uk.gov.nationalarchives.tdr.api.core.db.dao.FileDao.files
```

Comment:
consignments is an used import for can probably be removed

Reply:
Fixed.